intl/msg_hash_ar.c

```diff
@@ -25,7 +25,7 @@
 #include "../configuration.h"
 #include "../verbosity.h"
 
-#if defined(_MSC_VER) && !defined(_XBOX)
+#if (_MSC_VER >= 1500 && _MSC_VER < 1900) && !defined(_XBOX)
```

Comment:
you still need to check if _MSC_VER is defined first.
also, is the upper bound necessary ?

Reply:
`#pragma execution_character_set("utf-8")` is a hack that Microsoft introduced in MSVC 2008, or `_MSC_VER >= 1500`.  It was deprecated as of MSVC 2015, or `_MSC_VER < 1900`, and causes problems if combined with the /utf-8 switch which replaced it.